dts/arm/renesas/gen3/rcar_gen3_cr7.dtsi

```diff
@@ -70,6 +71,76 @@
 			reg = <0xe6060000 0x50c>;
 		};
 
+		pwm0: pwm@e6e30000 {
+			compatible = "renesas,rcar-pwm";
+			reg = <0xe6e30000 0x8>;
+			#pwm-cells = <2>;
+			clocks = <&cpg CPG_MOD 523>,
+				<&cpg CPG_CORE CPG_CORE_CLK_S0D12>;
+			status = "disabled";
+			label = "pwm0";
+		};
```

Comment:
Nits, Linux is describing it in this order:
- compatible
- reg
- clocks
- pwm-cells
- status

Reply:
https://github.com/torvalds/linux/blob/master/arch/arm64/boot/dts/renesas/r8a77951.dtsi

Reply:
Corrected, thanks. Now follows the Linux pwm-rcar organization.